src/components/LongGoalsComponent/Data.ts

```diff
@@ -0,0 +1,53 @@
+const data = [
+  {
+    id: '1',
```

Comment:
I'm not aware of what kind of id string is being used at the database wether it is "id" Or "_id" So just make sure to keep it in sync if in future you want to get this data from the backend.

Reply:
> I'm not aware of what kind of id string is being used at the database wether it is "id" Or "_id" So just make sure to keep it in sync if in future you want to get this data from the backend.

 I have just followed the format that is already used in the project. I'll do the changes